website/docs/r/cloud_provider_snapshot_backup_policy.html.markdown

```diff
@@ -0,0 +1,118 @@
+  project_id   = mongodbatlas_cluster.my_cluster.project_id
+  cluster_name = mongodbatlas_cluster.my_cluster.name
+
+  reference_hour_of_day    = 3
+  reference_minute_of_hour = 45
+  restore_window_days      = 4
+
+
+  policies {
+    id = mongodbatlas_cluster.my_cluster.snapshot_backup_policy.0.policies.0.id
+
+    policy_item {
+      id                 = mongodbatlas_cluster.my_cluster.snapshot_backup_policy.0.policies.0.policy_item.0.id
+      frequency_interval = 1
+      frequency_type     = "hourly"
+      retention_unit     = "days"
+      retention_value    = 1
+    }
+    policy_item {
+      id                 = mongodbatlas_cluster.my_cluster.snapshot_backup_policy.0.policies.0.policy_item.1.id
+      frequency_interval = 1
+      frequency_type     = "daily"
+      retention_unit     = "days"
+      retention_value    = 2
+    }
+    policy_item {
+      id                 = mongodbatlas_cluster.my_cluster.snapshot_backup_policy.0.policies.0.policy_item.2.id
+      frequency_interval = 4
+      frequency_type     = "weekly"
+      retention_unit     = "weeks"
+      retention_value    = 3
+    }
+    policy_item {
+      id                 = mongodbatlas_cluster.my_cluster.snapshot_backup_policy.0.policies.0.policy_item.3.id
+      frequency_interval = 5
+      frequency_type     = "monthly"
+      retention_unit     = "months"
+      retention_value    = 4
+    }
+  }
+}
+
+data "mongodbatlas_cloud_provider_snapshot_backup_policy" "test" {
+  project_id   = mongodbatlas_cloud_provider_snapshot_backup_policy.test.project_id
+  cluster_name = mongodbatlas_cloud_provider_snapshot_backup_policy.test.cluster_name
+}
+```
+
+## Argument Reference
+
+* `project_id` - (Required) The unique identifier of the project for the Atlas cluster.
+* `cluster_name` - (Required) The name of the Atlas cluster that contains the snapshots backup policy you want to retrieve.
+* `reference_hour_of_day` - (Optional) UTC Hour of day between 0 and 23, inclusive, representing which hour of the day that Atlas takes snapshots for backup policy items.
+* `reference_minute_of_hour` - (Optional) UTC Minutes after referenceHourOfDay that Atlas takes snapshots for backup policy items. Must be between 0 and 59, inclusive.
+* `restore_window_days` - (Optional) Number of days back in time you can restore to with point-in-time accuracy. Must be a positive, non-zero integer.
+* `update_snapshots` - (Optional) Specify true to apply the retention changes in the updated backup policy to snapshots that Atlas took previously.
+
+### Policies
+* `policies` - (Required) Contains a document for each backup policy item in the desired updated backup policy.
+* `policies.#.id` - (Required) Unique identifier of the backup policy that you want to update.
+
+#### Policy Item
+* `policies.#.policy_item` - (Required) Array of backup policy items.
+* `policies.#.policy_item.#.id` - (Required) Unique identifier of the backup policy item.
```

Comment:
Similar to above: 

(Required) Unique identifier of the backup policy item.    `policies.#.policy_item.#.id` is a value obtained via the mongodbatlas_cluster resource.  `provider_backup_enabled` of the mongodbatlas_cluster resource must be set to true.   See the example above for how to refer to the mongodbatlas_cluster resource for`policies.#.policy_item.#.id`

Reply:
I think this may have gotten resolved prematurely?

Reply:
I'm sorry I lost on this topic :S

Reply:
Sorry should have been more specific - One suggestion got added, the other did not so I reopened this comment to ensure both get updated.  I'll double check the latest commit too in case I missed it.

Reply:
okay Thank you so much!